app-creator/src/widgets/ui-label/ui-label.ts

```diff
@@ -51,18 +47,27 @@ export class Label extends LitElement {
   @property({ type: String })
   type = 'paragraph';
 
+  convertToStyleString(style: { [key: string]: string }) {
+    return Object.keys(style).reduce(
+      (styleString, key) => styleString + `${key}: ${style[key]};`,
```

Comment:
Use a map instead and join with`\n`, in order to keep the formatting on the DOM a little prettier.

Reply:
Or, if this function isn't being used, remove it.